notification-server/Cargo.toml

```
@@ -33,6 +33,8 @@ tracing = { version = "0.1" }
 tracing-subscriber = { version = "0.3" }
 
 [dev-dependencies]
+async-trait = "0.1.81"
+mockito = "1.4.0"
```

```suggestion
async-trait = "0.1"
mockito = "1.4"
```

Unless there is a reason for specifying the patch?